examples/jsm/postprocessing/ShaderPass.d.ts

```diff
@@ -1,15 +1,15 @@
 import {
-	Material
+	ShaderMaterial
 } from '../../../src/Three';
 
 import { Pass } from './Pass';
 
 export class ShaderPass extends Pass {
 
-	constructor( shader: object, textureID?: string );
+	constructor(shader: object, textureID?: string);
```

Comment:
There is no need to remove the white space.

Reply:
> There is no need to remove the white space.

Yes. you are right.